packages/ui/src/components/va-option-list/VaOptionList.vue

```diff
@@ -150,8 +142,16 @@ export default defineComponent({
 
     const { computedError, computedErrorMessages } = useValidation(props, emit, reset, focus)
 
+    const computedProps = computed(() => ({
+      ...pick(props, ['name', 'color', 'readonly', 'leftLabel']),
+    }))
+
+    const addRef = (el: HTMLInputElement, index: number) => {
```

Comment:
Take a look at `packages/ui/src/composables/useArrayRefs.ts`

Reply:
Thank you. Fine! Didn't notice that we have such a hook!